### PR TITLE
enable some more clangarm64 packages

### PR DESCRIPTION
**Pull request by kmilos**

### CODE REVIEW

File: mingw-w64-discount/PKGBUILD [hunk 1]
```diff
@@ -3,11 +3,11 @@
 _realname=discount
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=2.2.7
+pkgver=2.2.7b
```
Comment (jeremyd2019) [inline mingw-w64-discount/PKGBUILD:6]: Why are you downgrading? 😛 
```
$ vercmp 2.2.7 2.2.8
-1

$ vercmp 2.2.7 2.2.7b
1
```

Alternative:
```suggestion
pkgver=2.2.7.b
```

```
$ vercmp 2.2.7 2.2.7.b
-1
```

**Comment by jeremyd2019** [comment]
Failed ARM64 CI run: https://github.com/msys2-arm/MINGW-packages/runs/5890701095?check_suite_focus=true

File: mingw-w64-discount/PKGBUILD (hunk 1 above)
Comment (kmilos) [reply mingw-w64-discount/PKGBUILD]: Grr... Reverting the bump, the change is not significant anyway...

**Comment by kmilos** [comment]
> Failed ARM64 CI run

Thanks, let's leave drmingw out for another day then.

**Comment by kmilos** [comment]
I guess I'll have to rebase again once https://github.com/msys2/MINGW-packages/pull/11404 gets merged, but it would still be good to know if libwmf and graphicsmagick build...

**Comment by jeremyd2019** [comment]
Successful ARM64 CI run: https://github.com/msys2-arm/MINGW-packages/runs/6071809625?check_suite_focus=true

**Pull request merged**